src/MainPage.js

```diff
@@ -109,7 +105,7 @@ const GeolocationButton = styled(Button)`
     margin-bottom: 20px;
 `
 
-const GpsIconStyled = styled(GpsIcon)`
+const GpsIconSized = styled(GpsIcon)`
```

Comment:
What do you mean by ``sized`` in ``GpsIconSized``?  minimized or maximized ? Or sth. else? Is a bit confusing when you first read it.

Reply:
You're right, gonna change it to GpsIconSmall :D